public/js/pelagosNoty.js

```diff
@@ -28,4 +28,4 @@
             }
         });
     }
-}(jQuery));
\ No newline at end of file
+}(jQuery));
```

Comment:
New line

Reply:
Doesn't it have one right now? I didn't before...